core/templates/pages/exploration-player-page/learner-experience/tutor-card.directive.ts

```diff
@@ -229,6 +228,7 @@ angular.module('oppia').directive('tutorCard', [
             $scope.profilePicture = UrlInterpolationService.getStaticImageUrl(
               '/avatar/user_blue_72px.png');
 
+            var _editorPreviewMode = ContextService.isInExplorationEditorPage();
```

Comment:
Why this change?

Reply:
currently _editorPreviewMode is initialized with the initialization of the component itself, which makes it hard to test the parity check in the below if block. So, I've shifted this part of the code to `onInit` function.
(It is the only place where `_editorPreviewMode` is used)

Reply:
Ah ok thanks for explaining!